src/Ouroboros/Network/MonadClass/MonadConc.hs

```diff
@@ -38,3 +39,15 @@ newEmptyMVar = newEmptyNamedMVar Nothing
 
 newMVar :: MonadConc m => a -> m (MVar m a)
 newMVar = newNamedMVar Nothing
+
+instance MonadConc IO where
```

Comment:
Let's delete this, as it's unclear what to do with these "names" for the `MVar`s.